packages/node/src/indexer/indexer.manager.ts

```diff
@@ -77,8 +80,9 @@ export class IndexerManager {
     operationHash: Uint8Array;
     reindexBlockHeight: null;
   }> {
-    const { blockHeight } = blockContent;
+    const { block, blockHeight } = blockContent;
     let dynamicDsCreated = false;
+    let reindexBlockHeight = null;
```

Comment:
This also needs to be updated https://github.com/subquery/subql-ethereum/blob/3e91e131aaaa61b24170d9faa12b7b2a27207069/packages/node/src/indexer/indexer.manager.ts#L176